content/en/docs/reference/glossary/day-2-operation.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Day 2 Operations
+test: n/a
+---
+The period after initial installation and configuration where a system must be reliable, stable, secure and easily upgradeable.
```

Comment:
cc @nrjpoddar @shamsher31

I added an initial cut of what Day 2 Operations means. I'm not sure if this is enough context; for instance, we don't specify Day 1 operations. This is important to clarify as this is an important issue for the 2021 Roadmap which the TOC approved of today.

Reply:
You can link to an external blog/guide about what Day 2 operations mean instead of defining it ourselves.

Reply:
Added link to dzone